rules/kubecf/image_list.tmpl.rb

```diff
@@ -256,6 +256,12 @@ def images
 permutations = [true, false].repeated_permutation(features.size)
                             .map { |v| features.zip(v).to_h }
 
+# Running through all permutations takes very long, and makes it impractical to
+# add additional feature flags. Some flag combinations also could throw errors.
+# So far running with features either all enabled or all disabled generates
+# the complete set of used images and is nearly instantaneous.
+permutations = [ features.to_h{ |x| [x, false] }, features.to_h{ |x| [x, true] } ]
+
```

Comment:
Is line 256 (just before the added block) still needed ? I do not believe so, as the new block overrides it.

Reply:
It isn't needed, and I've removed it now; the implementation remains accessible in the git history...